openfl-tutorials/interactive_api/Tensorflow_CIFAR_tfdata/workspace/Tensorflow_CIFAR.ipynb

```diff
@@ -0,0 +1,395 @@
+    "\n",
+    "        # Update training metric.\n",
+    "        train_acc_metric.update_state(y, logits)\n",
+    "        pbar.update(step+1, \n",
+    "                    values={'loss': loss_value, 'acc': train_acc_metric.result()}.items())\n",
+    "        if warmup: break\n",
+    "    \n",
+    "    # Display metrics at the end of each epoch.\n",
+    "    train_acc = train_acc_metric.result()\n",
+    "    print(\"Training acc over epoch: %.4f\" % (float(train_acc),))\n",
+    "\n",
+    "    # Reset training metrics at the end of each epoch\n",
+    "    train_acc_metric.reset_states()\n",
+    "    return {'train_acc': train_acc,}\n",
+    "\n",
+    "\n",
+    "@TI.register_fl_task(model='model', data_loader='dataset', device='device')     \n",
+    "def validate(model, dataset, device):\n",
+    "    # Run a validation loop at the end of each epoch.\n",
+    "    for x, y in dataset:\n",
+    "        logits = model(x, training=False)\n",
+    "        # Update val metrics\n",
+    "        val_acc_metric.update_state(y, logits)\n",
+    "    val_acc = val_acc_metric.result()\n",
+    "    val_acc_metric.reset_states()\n",
+    "    print(\"Validation acc: %.4f\" % (float(val_acc),))\n",
+    "            \n",
+    "    return {'validation_accuracy': val_acc,}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f0ebf2d",
+   "metadata": {},
+   "source": [
+    "## Time to start a federated learning experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d41b7896",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an experimnet in federation\n",
+    "experiment_name = 'cifar10_experiment'\n",
+    "fl_experiment = FLExperiment(federation=federation, experiment_name=experiment_name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "41b44de9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The following command zips the workspace and python requirements to be transfered to collaborator nodes\n",
+    "ROUNDS_TO_TRAIN = 10\n",
+    "fl_experiment.start(model_provider=MI, \n",
+    "                   task_keeper=TI,\n",
+    "                   data_loader=fed_dataset,\n",
+    "                   rounds_to_train=ROUNDS_TO_TRAIN,\n",
+    "                   opt_treatment='CONTINUE_GLOBAL')\n",
+    "fl_experiment.stream_metrics()"
+   ]
+  }
+ ],
+ "metadata": {
+  "interpreter": {
+   "hash": "f82a63373a71051274245dbf52f7a790e1979bab025fdff4da684b10eb9978bd"
+  },
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Please clear metadata from the notebook

Reply:
Should be OK I believe. The last cell metadata is non-identifiable and wasn't cleared even upon clearing cell outputs. Marking it as resolved.

Open to alternative ways.